pwndbg/lib/functions_data.py

```diff
@@ -28479,7 +28480,13 @@ def load_functions():
             Argument(type="void", derefcnt=1, name="addr"),
             Argument(type="size_t", derefcnt=0, name="len"),
             Argument(type="int", derefcnt=0, name="prot"),
-            Argument(type="int", derefcnt=0, name="flags"),
+            Argument(type="int", derefcnt=0, name="flags", flags=[
```

Comment:
Should always be a tuple instead of a list

Reply:
I switched the type everywhere to `tuple[Flag, ...]`. I think that is what you're asking for?